sbi/inference/potentials/likelihood_based_potential.py

```diff
@@ -115,6 +116,52 @@ def __call__(self, theta: Tensor, track_gradients: bool = True) -> Tensor:
                 )
             return log_likelihood_batches + self.prior.log_prob(theta)  # type: ignore
 
+    def condition_on_theta(
+        self, theta_condition: Tensor, dims_to_sample: List[int]
+    ) -> Callable:
+        """Returns a potential function conditioned on a subset of theta dimensions.
+
+        The condition is a part of theta, but is assumed to correspond to a batch of iid
```

Comment:
I am a bit confused by this docstring. What this function aims to get a potential for p(\theta_to_samples | x_1,...,x_n, \theta_not_ot_sample) no? I don't really get what this paragraph should tell me.